core/descriptors/src/org/jetbrains/kotlin/descriptors/impl/PackageViewDescriptorAccessor.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.descriptors.impl
+
+import org.jetbrains.kotlin.descriptors.ModuleCapability
+import org.jetbrains.kotlin.descriptors.PackageViewDescriptor
+import org.jetbrains.kotlin.name.FqName
+import org.jetbrains.kotlin.storage.StorageManager
+
+interface PackageViewDescriptorAccessor {
```

Comment:
The name like `PackageViewDescriptor` seems to be more clear to me, but I don't insist here